src/js/fs.js

```diff
@@ -263,26 +255,25 @@ fs.readFileSync = function(path) {
   return Buffer.concat(buffers);
 };
 
+fs.writeFile = function(path, outdata, callback) {
+  var data = ensureBuffer(outdata);
```

Comment:
This allows to write data that is allowed in Buffer constructor, or a Buffer itself. This ensures that you can pass a string and write a file or you can pass a Buffer and write that to a file.

Please refer to NodeJS documentation 

https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback

If we are creating a FS module that 'looks' and has an API like the NodeJS one, wee should to be consistent with it, as this will prevent confusion amongst developers using our IoT.js